mattermost-plugin/plugin.json

```diff
@@ -6,8 +6,8 @@
     "support_url": "https://github.com/mattermost/mattermost-plugin-focalboard/issues",
     "release_notes_url": "https://github.com/mattermost/mattermost-plugin-focalboard/releases/tag/v0.1.0",
     "icon_path": "assets/starter-template-icon.svg",
-    "version": "0.2.0",
-    "min_server_version": "5.12.0",
+    "version": "0.6.7",
+    "min_server_version": "5.35.0",
```

Comment:
It actually should be 5.36

Reply:
Right! server `master` has currently `5.35` so I left it here for testing. Will update!

Reply:
Done in e14730d